play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -81,9 +82,9 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
   }
 
   val routesFile = config.underlying.hasPath("play.http.router") match {
-    case false => "routes"
+    case false => "conf/routes"
     case true => config.getString("play.http.router") match {
-      case None => "routes"
+      case None => "conf/routes"
       case Some(value)=> value.replace(".Routes", ".routes")
```

Comment:
Great! Yes the value does not include the "conf" path but it should be added - see <https://github.com/playframework/playframework/blob/2.2.1/framework/skeletons/java-skel/conf/application.conf#L20>:

```
# Router
# ~~~~~
# Define the Router object to use for this application.
# This router will be looked up first when the application is starting up,
# so make sure this is the entry point.
# Furthermore, it's assumed your route file is named properly.
# So for an application router like `conf/my.application.Router`,
# you may need to define a router file `my.application.routes`.
# Default to Routes in the root package (and `conf/routes`)
# application.router=my.application.Routes
```